cumulusci.yml

```diff
@@ -25,7 +23,27 @@ tasks:
         class_path: cumulusci.tasks.salesforce.Deploy
         options:
             path: dev_config/delete
-            
+
+    execute_install_apex:
+        description: Runs most of the install script methods from STG_InstallScript class
+        class_path: cumulusci.tasks.anonymous_apex.AnonymousApexTask
+        options:
+            apex: >
+                UTIL_CustomSettingsFacade.getSettings();
+                STG_InstallScript.insertMappings();
+                STG_InstallScript.insertRelationshipLookups();
+                TDTM_Global_API.setTdtmConfig( TDTM_Global_API.getTdtmConfig() );
+
+    execute_install_apex_managed:
+        description: Runs most of the install script methods from STG_InstallScript class
+        class_path: cumulusci.tasks.anonymous_apex.AnonymousApexTask
+        options:
+            apex: >
+                hed.UTIL_CustomSettingsFacade.getSettings();
```

Comment:
UTIL_CustomSettingsFacade isn't global, so it can't be accessed in managed installations. are the below method calls still meaningful if you don't call getSettings first? I don't know the install script well.